systemTests/test-scenarios/given/basic_gridftp_environment.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import os
+import io
+import sys
+import time
+
+
+class BasicGridftpEnvironment:
+	def __init__(self, context):
+		self._context = context
+
+	def is_running(self):
+		os.system("globus-gridftp-server -control-interface 127.0.0.1 -aa -p 5000 &")
```

Comment:
we need to work on this a bit, this command starts a process in the background (the & symbol) and so can't be killed from within python as you don't know what the process ID is. I think if you tried to run the system tests twice then it would fail because port 5000 is still in use.

I'd generally like to move to running this from a script (which either me or Conor will write) and then this step would be robust to us changing how the process runs. This class would then just have to track the process id of the script and kill it when the test is done.

Reply:
there is a subprocess library that might be useful for this: https://stackoverflow.com/questions/19152067/execute-linux-command-and-get-pid